tests/config/test_claude_md.sh

```
@@ -10,85 +10,91 @@ test_claude_md_exists() {
         "CLAUDE.md should exist in repository root"
 }
 
-# Test CLAUDE.md structure - Updated for new pragmatic format
-test_claude_md_structure() {
+# Test CLAUDE.md effectiveness criteria
+test_claude_md_effectiveness() {
     local claude_file="$ORIGINAL_DIR/CLAUDE.md"
+    local line_count=$(wc -l < "$claude_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)